.github/workflows/storefront-deploy.yml

```diff
@@ -0,0 +1,23 @@
+name: Vercel Storefront Production Deployment
+env:
+  VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+  VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
```

Comment:
Maybe we should call this secret something like `VERCEL_PROJECT_ID_STOREFRONT`? Since it seems like project = 1 site/app in vercel. Then its clear which projectid this is and we keep the door open for hosting other sites on vercel :)

Reply:
Good idea!
* Removed `VERCEL_PROJECT_ID` and added `VERCEL_PROJECT_ID_STOREFRONT` to secrets.
* Changed reference in the action files.
* Tested deployment again and works.